src/useFetch.ts

```diff
@@ -83,13 +83,17 @@ function useFetch<TData = any>(...args: UseFetchArgs): UseFetch<TData> {
       const isCached = cache.has(requestID)
       const cachedData = cache.get(requestID)
       if (isCached && cachePolicy === CACHE_FIRST) {
+        setLoading(true)
```

Comment:
We shouldn't need this. If the data is cached it should be immediate. After this get's merged I have a change that instead of caching the data, we cache the `response` which fixes other problems. Regardless, we don't need to modify state if it's cached.